### DISCUSSION TITLE
Cydia roles in settings

### DISCUSSION THREAD

**Discussion by worldlydev** [wstyres/Zebra#678] [Ideas]
**Describe the solution you'd like**

https://www.reddit.com/r/jailbreak/comments/de5wk8/release_cyrole_bring_back_the_cydia_roles/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by wstyres** [comment]
This is something I completely forgot about, we can do it in a future version. I'm thinking about a intro/setup screen and that setting would likely appear there as well.

**Comment by worldlydev** [comment]
> This is something I completely forgot about, we can do it in a future version. I'm thinking about a intro/setup screen and that setting would likely appear there as well.

Cool. Maybe something along the lines of this: https://twitter.com/DaveWijk/status/1186759781803085824?s=20 . Could include repo import, currently existing settings, and perhaps #680.

**Comment by worldlydev** [comment]
Update: When this is implemented you could rename the different roles. The old names seem outdated.

Still use the same tag system, just new selectable names in Zebra. User and Developer are fine. I just feel Hacker should be renamed to something that is more inline with the packages that actually get shown.

Perhaps like this:
![C618724A-87FC-42FD-B2BC-5186371FF2B2](https://user-images.githubusercontent.com/51668635/83779208-f2a84380-a68b-11ea-914c-ecc4952fe61e.jpeg)
Power User is a more fitting name than hacker.
